commit b300e47c3b374cd2dfb3e69ff0dc85da03e257b2 (refs/changes/62/4462/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-06-29T20:03:00+00:00 (1 year, 3 months ago)
    
    TRITON-565 cnapi README testing section could be a bit more helpful

diff --git a/README.md b/README.md
index 7bf4627..2793e3a 100644
--- a/README.md
+++ b/README.md
@@ -44,3 +44,9 @@ The CNAPI_IP environment variable specifies the URL at which to point the test
 suite.  For example, this would test CNAPI installed on bh1-kvm6:
 
     CNAPI_IP=10.2.206.13 make test
+
+To test on a deployed CNAPI where the admin network is not directly accessible
+from your workspace, first ensure your changes are applied (either via `sdcadm`
+or the `tools/rsync-to` script) and then run:
+
+    COAL=root@<headnode external IP> make test-coal
